community/front-end/ofe/infrastructure_files/workbench_tf/google/variables.tf

```diff
@@ -56,16 +56,56 @@ variable "folder_id" {
   default     = ""
 }
 
+# Deprecated, replaced by instance_image
+# tflint-ignore: terraform_unused_declarations
 variable "image_family" {
   description = "Image of the AI notebook."
   type        = string
   default     = "tf-latest-cpu"
+
+  validation {
+    condition     = var.image_family == null
+    error_message = "The 'image_family' setting is deprecated, please use 'instance-image' with the sub-settings 'project' and 'family' or 'name'."
```

Comment:
You should either say `var.instance_image` or `instance_image`, but either way use an underscore. I would also describe the "sub-settings" as "fields" instead.

Reply:
Done